bridge:
  - factory: mqtt_bridge.bridge:RosToMqttBridge
    msg_type: std_msgs.msg:String
    topic_from: /machine/logger
    topic_to: logger/machine
  - factory: mqtt_bridge.bridge:MqttToRosBridge
    msg_type: geometry_msgs.msg:Twist
    topic_from: aws/iot/available
    topic_to: /aws/iot/available
  - factory: mqtt_bridge.bridge:RosToMqttBridge
    msg_type: std_msgs.msg:Bool
    topic_from: /machine/available
    topic_to: logger/machine/available
  - factory: mqtt_bridge.bridge:MqttToRosBridge
    msg_type: std_msgs.msg:Bool
    topic_from: aws/sumerian/available
    topic_to: /aws/sumerian/available
mqtt: 
  connection: 
    host: a1ahmtv2bnhf50-ats.iot.us-east-1.amazonaws.com
    port: 8883
    keepalive: 60
    mode: both
  client: 
    clientId: basicPubSub
    protocol: 4
    websocket : False
tls: 
  rootCA: /home/dftossem/Documents/aws/credentials/dummy/root-CA.crt
  certKey: /home/dftossem/Documents/aws/credentials/dummy/c35c934f0c-certificate.pem.crt
  privKey: /home/dftossem/Documents/aws/credentials/dummy/c35c934f0c-private.pem.key
